pkg/pilot/cassandra/v3/pilot.go

```diff
@@ -39,6 +41,22 @@ func NewPilot(opts *PilotOptions) (*Pilot, error) {
 		nodeTool:            opts.nodeTool,
 	}
 
+	// hack to test the seedprovider works in k8s, this should be moved somewhere better
+	cfgPath := "/etc/cassandra/cassandra.yaml"
+	read, err := ioutil.ReadFile(cfgPath)
+	if err != nil {
+		return nil, err
+	}
+
+	newContents := strings.Replace(string(read),
+		"org.apache.cassandra.locator.SimpleSeedProvider",
+		"io.k8s.cassandra.KubernetesSeedProvider", -1)
```

Comment:
This is a good idea, otherwise we're relying on the config file from the cassandra container being in a certain state when we start making these modifications. Maybe we should address this in https://github.com/jetstack/navigator/issues/251, though?

Reply:
> Maybe we should address this in #251, though?

That's fine by me.